compiler.go

```diff
@@ -640,6 +640,12 @@ func (c *Compiler) SetImportDir(dir string) {
 	c.importDir = dir
 }
 
+// SetImportExt sets the custom extension name of the source file for
+// loading local module files.
+func (c *Compiler) SetImportExt(ext string) {
```

Comment:
Maybe some basic validations here to make sure the extension isn't empty and is an actual extension.

Reply:
Indeed! I will add the basic validation to the method and test for it.

Reply:
I added a simple validation. It now returns an error if any given extension is not valid.

```diff
- func (c *Compiler) SetImportExt(ext string) {
+ func (c *Compiler) SetImportFileExt(exts ...string) error {
````

Also, I changed the `Compiler`'s `importExt` field to `importFileExt`. Due to make it clear it is an extension for the source file and not an extension of tengo, such as a plug-in.